docs/hard-fork-upgrade/prerequisites-and-planning.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Archive migration prerequisites
+sidebar_label: Archive migration prerequisites
+hide_title: true
+description: Mina protocol havily relies on its archive node. It is crucial to properly migrate archives from Mainnet to Berkeley.
+keywords:
+  - hard fork
+  - upgrade
+  - archive migration
+  - planning 
+  - prerequisites
+  - mina archive node
+  - archive node
+---
+
+# Migrating Mainnet Database to Berkeley
+
+Before you start planning the migration of your Mainnet database into the Berkeley version of the Mina network, read this document in entirety.
+
+## Overview
+
+Mainnet database requires to be migrated in two steps. Each step uses an application that is shipped by Mina Foundation and o1Labs teams:
+
+- **berkeley-migration**
+
+  The berkeley-migration application migrates as much data as possible from the Mainnet database and downloads precomputed blocks to get the window density data. 
+  
+  This application runs against the Mainnet and the new database.
+
+- **replayer in migration mode**
+
+  The replayer application is an existing appliction, but enhanced with a new mode: migration mode. Replayer in migration mode analyze The transactions in the partially migrated database and populates the `accounts_accessed` and `accounts_created` tables. This application also performs the checks performed by the standard replayer, except that ledger hashes are not checked because the hard fork ledger has greater depth, which results in different hashes. This application runs only against the new database.
```

Comment:
typo - application - word 7

Reply:
hey @halsaphi  you are a great reviewer! Do you know about GitHub suggestions? see https://github.com/o1-labs/docs2/blob/main/CONTRIBUTING.md#reviewing-technical-content-prs